src/lib/serviceRoutinesV2/postEntities.cpp

```diff
@@ -68,9 +69,13 @@ std::string postEntities
   // 02. Call standard op postUpdateContext
   postUpdateContext(ciP, components, compV, parseDataP);
 
+  HttpStatusCode rcode = parseDataP->upcrs.res.contextElementResponseVector[0]->statusCode.code;
+
+  // Operation answer
+  std::string answer;
```

Comment:
Same as before, comment on same line and variable alignment.
Actually, is this comment even necessary?
At least I don't get any wiser with the comment ...

Normally (not always - some are really complex), if a variable needs an explanation, then the name of the variable is not good and should be changed.